### ISSUE TITLE
FB.AppRequest Crash on Android (FB SDK 11.0)

### ISSUE THREAD

**Issue by dmichalke** [facebook/facebook-sdk-for-unity#564]
## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-sdk-for-unity/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [ ] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

- Unity Editor Version: `2021.1.11`
- Unity Facebook SDK Version: `11.0`
- Installation Platform & Version: `[Android]` version `4.4`

## Goals

Invite Friends to the app with FB.AppRequest

## Expected Results

SDK displays FB dialog that shows friends, like it did in 7.2 (the FB version we used before) when running:
FB.AppRequest(message, to, null, null, null, data, title, result => onComplete.Invoke(result.ToFacebookResult()));

## Actual Results

app crashes (test device Sony Xperia L4), check screenshot

![Screenshot_1](https://user-images.githubusercontent.com/86662384/123790211-59734d00-d8de-11eb-8bc9-a393e8cb3973.png)

**Comment by gameplayer05** [comment]
Yes, even i am also facing same crash problem with Facebook SDK 11.0.0

**Comment by spielepalast** [comment]
what is the current status?

**Comment by paradizIscool** [comment]
I have the same issue in 9.2
